src/controllers/authController.js

```diff
@@ -0,0 +1,188 @@
+const User = require('../models/user.js');
+const userService = require('../services/userService.js');
+const tokenService = require('../services/tokenService.js');
+const emailService = require('../services/emailService.js');
+const jwtService = require('../services/jwtService.js');
+const ApiError = require('../exeptions/apiError.js');
+const bcrypt = require('bcrypt');
+const { v4: uuidv4 } = require('uuid');
+
+const validateEmail = (value) => {
+  if (!value) {
+    return 'Email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(value)) {
+    return 'Email is not valid';
+  }
+};
+
+const validatePassword = (value) => {
+  if (!value) {
+    return 'Password is required';
+  }
+
+  if (value.length < 6) {
+    return 'At least 6 characters';
+  }
+};
```

Comment:
The `validatePassword` function should return a success message or `null` when the password is valid. Currently, it returns nothing, which might lead to unexpected behavior.